File: Assignment 29/program4.c
#include<stdio.h>

void pattern(int iRow,int iCol)
{
    int i=0,j=0;
    
    for(i=1;i<=iRow;i++)
    {
        
        for(j=1;j<=iCol;j++)
        {    
             int k=0;
             if(i%2==0)
             {
                for(k=-1;k>=-iCol;k--)
                {
                  printf("%d\t",k);
                }
                break;
                 
             }
           
             else
            {
                printf("%d\t",j);
            }
            

        }
       printf("\n");
       
    }
    printf("\n");

}

int main()
{

    int iValue1=0,iValue2=0;

    printf("Enter the Number of Rows:");
    scanf("%d",&iValue1);

    printf("Enter the Number of Columns:");
    scanf("%d",&iValue2);

    pattern(iValue1,iValue2);


    return 0;
}

File: Assignment 27/program4.java
import java.util.*;

class Pattern
{ 
    public void Display(int iRow,int iCol)
    {
        int i=0,j=0;
        

        for(i=1;i<=iRow;i++)
        {
            for(j=1;j<=iCol;j++)
            {
                if(j%2==1)
                {
                    System.out.print(" * ");
                }
                else
                {
                    System.out.print(" # ");
                }
               

            }
             System.out.println();

        
        }
        System.out.println();

    }

}

class program4
{
    public static void main(String A[])
    { 
        int iValue1=0,iValue2=0;

        Scanner sobj=new Scanner(System.in);
        System.out.println("Enetr no of Rows:");
        iValue1=sobj.nextInt();

        System.out.println("Enetr no of Columns:");
        iValue2=sobj.nextInt();

        Pattern pobj=new Pattern();
        pobj.Display(iValue1,iValue2);

    }
}